data/json/items/book/fabrication.json

```
@@ -456,8 +456,8 @@
     "skill": "fabrication",
     "max_level": 3,
     "proficiencies": [
-      { "proficiency": "prof_carpentry_basic", "time_factor": 0.5, "fail_factor": 0.4, "include_prereqs": true },
-      { "proficiency": "prof_plumbing", "time_factor": 0.5, "fail_factor": 0.6, "include_prereqs": true }
+      { "proficiency": "prof_carpentry_basic", "time_factor": 0.1, "fail_factor": 0.4, "include_prereqs": true },
+      { "proficiency": "prof_plumbing", "time_factor": 0.1, "fail_factor": 0.2, "include_prereqs": true }
```

```suggestion
      { "proficiency": "prof_plumbing", "time_factor": 0.1, "fail_factor": 0.3, "include_prereqs": true }
```
More similar to values for 101_carpentry.